soc/sifive/sifive_freedom/fu500/Kconfig.soc

```diff
@@ -5,12 +5,20 @@ config SOC_SERIES_SIFIVE_FREEDOM_FU500
 	bool
 	select SOC_FAMILY_SIFIVE_FREEDOM
 
-config SOC_SERIES
-	default "fu500" if SOC_SERIES_SIFIVE_FREEDOM_FU500
-
 config SOC_SIFIVE_FREEDOM_FU540
 	bool
 	select SOC_SERIES_SIFIVE_FREEDOM_FU500
 
+config SOC_SIFIVE_FREEDOM_FU540_E51
+	bool
+	select SOC_SIFIVE_FREEDOM_FU540
+
+config SOC_SIFIVE_FREEDOM_FU540_U54
+	bool
+	select SOC_SIFIVE_FREEDOM_FU540
+
+config SOC_SERIES
+	default "fu500" if SOC_SERIES_SIFIVE_FREEDOM_FU500
```

Comment:
these need to match, so `SOC_SERIES_SIFIVE_FREEDOM_FU500` = `sifive_freedom_fu500`

Reply:
Not sure I follow - we're not changing this in this PR, just moving it a bit in the file. Plus, if we'd do the change from "fu500" to "sifive_freedom_fu500", then the whole path would have to change from `soc/sifive/sifive_freedom/fu500` to `soc/sifive/sifive_freedom/sifive_freedom_fu500`, which seems a bit redundant.